packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -1579,25 +1579,13 @@ describe('ReactDOMInput', () => {
       container,
     );
 
-    if (disableInputAttributeSyncing) {
-      expect(log).toEqual([
-        'set attribute type',
-        'set attribute min',
-        'set attribute max',
-        'set attribute step',
-        'set property value',
-      ]);
-    } else {
-      expect(log).toEqual([
-        'set attribute type',
-        'set attribute min',
-        'set attribute max',
-        'set attribute step',
-        'set property value',
-        'set attribute value',
-        'set attribute checked',
-      ]);
-    }
+    expect(log).toEqual([
```

Comment:
Why was this changed?

Reply:
It was a JSDOM workaround that works different in a browser: https://codesandbox.io/s/resets-value-of-datetime-input-to-fix-bugs-in-ios-safari-1ppwh

Reply:
This needs to go back to what it was before and we need to figure out why there's a difference. We can't just remove it.

Reply:
But it's not actual browser behavior. See linked codesandbox.